sparse/usr/bin/droid/droid-reclaim-memory.sh

```diff
@@ -4,21 +4,25 @@ while [ "$(wc -l < /proc/swaps)" -lt 2 ]; do
     sleep 1
 done
 
-TASKS=$(cat /sys/fs/cgroup/systemd/system.slice/*/cgroup.procs /sys/fs/cgroup/unified/system.slice/*/cgroup.procs /sys/fs/cgroup/system.slice/*/cgroup.procs)
+TASKS=$(cat /sys/fs/cgroup/systemd/system.slice/*/cgroup.procs /sys/fs/cgroup/unified/system.slice/*/cgroup.procs /sys/fs/cgroup/system.slice/*/cgroup.procs 2>/dev/null)
```

Comment:
Replace this with find and you can replace the `>/dev/null`:
```bash
find /sys/fs/cgroup -name cgroup.procs -exec cat {} \;
2
```

Reply:
Replaced with slightly modified version of find command you gave.